easybuild/easyblocks/a/advisor.py

```diff
@@ -46,6 +47,14 @@ def __init__(self, *args, **kwargs):
         else:
             self.subdir = 'advisor'
 
+    def prepare_step(self, *args, **kwargs):
+        """Since 2019u3 there is no license check."""
+        if LooseVersion(self.version) >= LooseVersion('2019_update3'):
```

Comment:
Same comment as for vtune, no need for the `else:`

Reply:
fixed, updated easyblock re-tested in [related easyconfig PR](https://github.com/easybuilders/easybuild-easyconfigs/pull/8140#issuecomment-486672811).